src/lib/utils/hooks/offer/useCheckTokenGatedOffer.ts

```diff
@@ -58,7 +59,7 @@ export default function useCheckTokenGatedOffer({
       }
 
       try {
-        const met = await core.checkTokenGatedCondition(condition, address);
+        const met = await core.checkTokenGatedCondition(offerId, address);
```

Comment:
Does this invocation refer to this method [here](https://github.com/bosonprotocol/core-components/blob/main/packages/react-kit/src/hooks/tokenGated/useCheckTokenGatedOffer.ts#L14)